src/interpreter/error/async-iterator-not-supported-error/async-iterator-not-supported-error.ts

```diff
@@ -6,6 +7,10 @@ import {IAsyncIteratorNotSupportedErrorOptions} from "./i-async-iterator-not-sup
  */
 export class AsyncIteratorNotSupportedError extends EvaluationError {
 	constructor({message = `It is not possible to evaluate an async iterator'`, typescript, environment}: IAsyncIteratorNotSupportedErrorOptions) {
-		super({message, environment, node: typescript.factory?.createEmptyStatement() ?? typescript.createEmptyStatement()});
+		super({
+			message,
+			environment,
+			node: typescript.factory?.createEmptyStatement() ?? (typescript as any as (typeof TS)["factory"]).createEmptyStatement()
```

Comment:
This is definitely me being nitpicky, but I'd prefer casting to `unknown` over `any`, and maybe a cleaner cast would be: `typescript.factory?.createEmptyStatement() ?? (typescript as unknown as TS.NodeFactory).createEmptyStatement()` instead :)